tracer/test/snapshots/NUnitEvpTests.SubmitTraces_packageVersion=all_evpVersionToRemove=null_expectedGzip=null.verified.txt

```diff
@@ -25,7 +25,7 @@
       env: integration_tests,
       error.msg: ErrorMessage,
       error.stack: StackTrace,
-      error.type: SetUpException,
+      error.type: System.Exception,
```

Comment:
Are these changes expected/desirable? 🤔 IS this a separate bug fix?

Reply:
Yeah, I created a new method to extract the actual ExceptionType (the split code) before this change I was filling that with a constant containing the origin of the exception. I think now is better.